feat(unlock-protocom-com) -  update images in unlock homepage carousel above the fold

kalidiagne opened the pull request:
<!--
Thank you for your contribution to Unlock Protocol!
-->

# Description
https://github.com/unlock-protocol/unlock/issues/9268

Update homepage carousel 

- [x] updated links
- [x] update images
- [x] update styling

<!--
Please include a summary of the change and which issue is fixed -include its number-. It's important that PRs connect to an existing issue, and we'll review this PR in part based on the content of that issue. Please also include relevant motivation and context.
-->

# Issues

<!-- This PR should fix or reference at least one existing issue ID. Add or delete as appropriate. -->

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

<!--
PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)
-->

## Release Note Draft Snippet

<!--

If relevant, please write a summary of your change that will be suitable for inclusion in the Release Notes for the next Unlock release.

-->

searchableguy commented:
You can remove unused fields from the data if they aren't going to be used.

One thing I would like you to fix is the current sliding mechanism which uses useEffect and result in janky behavior. 

Also make sure to apply styles for each item to be the same size (currently, it doesn't. Use object and w&h properties in tailwind)

ccarfi commented:
@kalidiagne — event ticketing image not showing?
<img width="1367" alt="Screen Shot 2022-07-19 at 6 41 04 AM" src="https://user-images.githubusercontent.com/214760/179666177-4fa09334-4a6e-4003-9707-3705d6009527.png">

kalidiagne commented:
> @kalidiagne — event ticketing image not showing? <img alt="Screen Shot 2022-07-19 at 6 41 04 AM" width="1367" src="https://user-images.githubusercontent.com/214760/179666177-4fa09334-4a6e-4003-9707-3705d6009527.png">

fixed! 
also removed all the links

julien51 commented:
@kalidiagne please do a final check from the netlify draft deploy

julien51 commented:
Also please implement @searchableguy suggestions

kalidiagne commented:
Ill merge and do the other small things in different PR
